From: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Date: Wed, 20 Oct 2010 19:13:15 +0000
Subject: wcb4xxp: Support beroNet BN4S0e PCI Express card
Bug: http://bugs.debian.org/600839
Origin: http://svn.asterisk.org/svn/dahdi/linux/branches/2.3@9455

Add extra PCI IDs to support "beroNet BN4S0e PCI Express 4x S0 Karte".

---
 drivers/dahdi/wcb4xxp/base.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/dahdi/wcb4xxp/base.c b/drivers/dahdi/wcb4xxp/base.c
index bb33d71..6a67c8d 100644
--- a/drivers/dahdi/wcb4xxp/base.c
+++ b/drivers/dahdi/wcb4xxp/base.c
@@ -2896,6 +2896,7 @@ static struct pci_device_id b4xx_ids[] __devinitdata =
 	{ 0x1397, 0x08b4, 0x1397, 0xb761, 0, 0, (unsigned long)&hfc2s_BN },
 	{ 0x1397, 0x08b4, 0x1397, 0xb560, 0, 0, (unsigned long)&hfc4s_BN },
 	{ 0x1397, 0x08b4, 0x1397, 0xb550, 0, 0, (unsigned long)&hfc4s_BN },
+	{ 0x1397, 0x08b4, 0x1397, 0xb762, 0, 0, (unsigned long)&hfc4s_BN },
 	{ 0x1397, 0x16b8, 0x1397, 0xb562, 0, 0, (unsigned long)&hfc8s_BN },
 	{ 0x1397, 0x16b8, 0x1397, 0xb56b, 0, 0, (unsigned long)&hfc8s_BN },
 	{ 0x1397, 0x08b4, 0x1397, 0xb540, 0, 0, (unsigned long)&hfc4s_SW },
-- 
1.7.1

